fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -59,7 +59,7 @@ where
                 ))
             })?,
         );
-        if pc < self.registers[RegId::IS] || pc >= self.registers[RegId::SSP] {
+        if pc < self.registers[RegId::IS] || pc >= self.registers[RegId::SP] {
```

Comment:
Note: this alone relaxes the stack data cannot be excutable -restriction.

Reply:
This should probably be reverted before merging, if we're not going to make use of is after all.

Reply:
Let's remove it from the current PR